Automattic/pocket-casts-android
# Use rectangular rounded buttons in widgets

## Description

This PR changes design of new widgets. Instead of rounded buttons we use square ones that scale in width. Corner size might change in the final design but it can be addressed in a followup PR.

Internal ref: Lkt7fuf9Nq3XvfAFPCfpTD-fi-1499_3734

## Testing Instructions

> [!note]
> Test this on a device with SDK >= 31. Rounding of corners below SDK 31 will be addressed in a separate PR.

1. Add widgets to the home screen.
2. Verify the rectangular design of buttons.

## Screenshots or Screencast 

| Filled | Empty | Medium resized |
| - | - | - |
| ![filled](https://github.com/Automattic/pocket-casts-android/assets/30936061/24ae0180-60b0-42d5-882b-239c16e09c40) | ![empty](https://github.com/Automattic/pocket-casts-android/assets/30936061/c7daafe3-6d85-4bb0-9564-cc60bdb191b3) | ![medium](https://github.com/Automattic/pocket-casts-android/assets/30936061/c690394e-ead9-4fe0-b1c4-022722561617) |

## Checklist

 
#### I have tested any UI changes...
